tools/spec_coverage/lib/spec_parser.dart

```diff
@@ -0,0 +1,61 @@
+import 'spec.dart';
+
+class SpecParser {
```

Comment:
OK, so this class depends in detail on the format of `spec.txt`. It's a good thing that this dependency is encapsulated to this class alone.